gating/pre_merge_test/run_deploy_mnaio.sh

```diff
@@ -109,11 +109,11 @@ env | grep RE_ | sed 's/^/export /' > /opt/rpc-openstack/RE_ENV
 # check if we're using artifacts or not
 if [[ ${RE_JOB_IMAGE} =~ no_artifacts$ ]]; then
   echo "export RPC_APT_ARTIFACT_ENABLED=no" >> /opt/rpc-openstack/RE_ENV
-  ${MNAIO_SSH} "apt-get -qq update; DEBIAN_FRONTEND=noninteractive apt-get -y upgrade"
+  ${MNAIO_SSH} "apt-get -qq update; DEBIAN_FRONTEND=noninteractive apt-get -y dist-upgrade"
```

Comment:
Given that this is just a host for the VM's, is this even necessary?

Reply:
It's actually running the command on the infra1 host which will be doing the deployment.

Reply:
Technically this will probably noop given the VMs are freshly provisioned.